src/codeu/chat/server/View.java

```diff
@@ -85,6 +85,85 @@ public ServerInfo getInfo() {
   @Override
   public Message findMessage(Uuid id) { return model.messageById().first(id); }
 
+  @Override
+  public Collection<String> userStatusUpdate(String name, Uuid owner) {
+
+    Collection<ConversationPayload> allConversations = all(model.conversationPayloadById());
+    Collection<String> contributions = new ArrayList<String>();
+
+    final User foundOwner = model.userById().first(owner);
+    final User foundUser = model.userByText().first(name);
+    
+    if(foundUser != null) {
```

Comment:
Also, one thing that might make your code a little easier to read here is unnesting your validation checks.

For example, right now it's:
if (condition) {
  // do stuff
} else {
 // error case
}

You could instead do this in the style of what's sometimes called "guard clauses", which look like this
if (errorCondition) {
  // print error
  return null or empty collection;
}

// do stuff

That way, the body of what you're trying to do doesn't end up nested a few layers deep.